ci-operator/config/operator-framework/community-operators/operator-framework-community-operators-master.yaml

```diff
@@ -0,0 +1,27 @@
+base_images:
+  cli:
+    name: "4.6"
+    namespace: ocp
+    tag: cli
+build_root:
+  project_image:
+    dockerfile_path: scripts/ci/Dockerfile.ci-operator
+resources:
+  '*':
+    requests:
+      cpu: 1000m
+      memory: 4Gi
+tag_specification:
+  name: "4.6"
+  namespace: ocp
+tests:
+- as: deploy-operator-on-openshift
+  commands: |
+    ls
+    /go/src/github.com/operator-framework/community-operators/scripts/ci/openshift-deploy.sh
```

Comment:
Please use the relative path (workdir will be the a working copy of the repo, but we do not guarantee the exact location)